Drift/config/services.yaml

```diff
@@ -0,0 +1,19 @@
+parameters:
+    app_path: "../.."
+
+# config/services.yaml
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+
+    App\:
+        resource: '%kernel.project_dir%/src/*'
+        exclude: '%kernel.project_dir%/src/{DependencyInjection,Entity,Migrations,Tests}'
```

Comment:
This block doesn't make sense, as including "everything" is never a good decision. I turned autowiring friendly as I discovered that it decreases so much the code and the configuration spent time, but I'm still against the initial skeleton preset configuration.